src/main/java/de/tum/in/www1/artemis/web/rest/dto/CourseManagementOverviewDTO.java

```diff
@@ -0,0 +1,28 @@
+package de.tum.in.www1.artemis.web.rest.dto;
+
+import java.util.List;
+
+import de.tum.in.www1.artemis.domain.Exercise;
+
+public class CourseManagementOverviewDTO {
```

Comment:
do we really need this DTO? If we send a List of exercises, those include the course anyway. I don't see an advantage to store the courseId another time

Reply:
We send an list of those DTOs. The course id is to be able to identify the course it belongs to.